dataset/startup/api_registry.py

```diff
@@ -16,10 +16,9 @@
 from fastapi import FastAPI
 
 from dataset.components.dataset.views import router as dataset_routers
-from dataset.routers import api_root
+from dataset.components.health.views import router as health_routers
```

Comment:
I would suggest to use singular since it's once router with routes.
```suggestion
from dataset.components.health.views import router as health_router
```